cmd/keytransparency-signer/main.go

```diff
@@ -108,20 +82,11 @@ func main() {
 	factory := transaction.NewFactory(sqldb)
 
 	// Connect to map server.
-	var tmap trillian.TrillianMapClient
-	if *mapURL != "" {
-		mconn, err := grpc.Dial(*mapURL, grpc.WithInsecure())
-		if err != nil {
-			glog.Exitf("grpc.Dial(%v): %v", *mapURL, err)
-		}
-		tmap = trillian.NewTrillianMapClient(mconn)
-	} else {
-		var err error
-		tmap, err = newMapServer(context.Background(), sqldb, factory)
-		if err != nil {
-			glog.Exitf("newMapServer: %v", err)
-		}
+	mconn, err := grpc.Dial(*mapURL, grpc.WithInsecure())
```

Comment:
Yes, Trillian is supposed to be a backend connection, not exposed to the internet so at the moment it's unencrypted.